tools/test-proxy/Azure.Sdk.Tools.TestProxy.Tests/RecordSessionTests.cs

```diff
@@ -541,6 +541,43 @@ public void ContentLengthResetToZeroOnGetRequestWithEmptyBody()
             ContentLengthUpdatedCorrectlyOnEmptyBody(isHeadRequest: false);
         }
 
+        [Fact]
```

Comment:
SHould we also add a case to https://github.com/Azure/azure-sdk-tools/blob/515ec30f92ff57f8497a77fec6c6dd0c2d9ad184/tools/test-proxy/Azure.Sdk.Tools.TestProxy.Tests/RecordSessionTests.cs#L34 ?

Reply:
If we don't already have them, I think we should also add tests for the following responses:

1. No body (`content-length: 0`)
2. Body contains other literal JSON values like `"foo"` (arbitrary string), `19` (arbitrary number), `true`, or `false`
3. Body contains empty object `{}` or empty array `[]`
3. Body contains JSON string `"null"` (as opposed to JSON null-value `null`)